examples/jdbc-client-batch-update/jdbc_client_batch_update.bal

```diff
@@ -0,0 +1,104 @@
+import ballerina/io;
+import ballerina/jdbc;
+
+// Client endpoint for MySQL database. This client endpoint can be used with any jdbc
+// supported database by providing the corresponding jdbc url.
+endpoint jdbc:Client testDB {
+    url: "jdbc:mysql://localhost:3306/testdb",
+    username: "test",
+    password: "test",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+
+// This is the type created to represent data row.
+type Student record {
+    int id;
+    int age;
+    string name;
+};
+
+type myBatchType int|string;
+
+public function main() {
+    // Creates a table using the update operation. If the DDL
+    // statement execution is successful, the `update` operation returns 0.
+    io:println("The update operation - Creating a table:");
+    var ret = testDB->update("CREATE TABLE student(id INT AUTO_INCREMENT,
+                         age INT, name VARCHAR(255), PRIMARY KEY (id))");
+    handleUpdate(ret, "Create student table");
+
+    json jsonMsg = {
+        "student": [{
+            "firstname": "Peter",
+            "age": 10
+        }, {
+            "firstname": "John",
+            "age": 15
+        }, {
+            "firstname": "James",
+            "age": 12
+        }]
+    };
+
+    // Prepare the data batches.
+    int datalen = lengthof jsonMsg.student;
+    myBatchType[][] dataBatch;
+    int i = 0;
+
+    foreach (studentData in jsonMsg.student) {
+        string name = studentData.firstname.toString();
+        int age = check <int>studentData.age;
+
+        myBatchType[] dataRow = [age, name];
+        dataBatch[i] = dataRow;
+        i = i + 1;
+    }
+    // A batch of data can be inserted using the `batchUpdate` operation. The number
+    // of inserted rows for each insert in the batch is returned as an array.
+    var retBatch = testDB->batchUpdate("Insert into student
```

Comment:
Shall we stick to either upper or lower case for queries? In some places we have used upper case.